stream-applications-build/pom.xml

```diff
@@ -50,6 +50,7 @@
         <mavenThreads>1</mavenThreads>
         <commons-compress.version>1.26.2</commons-compress.version>
         <commons-lang3.version>3.17.0</commons-lang3.version>
+		<okhttp3.version>4.12.0</okhttp3.version>
```

Comment:
Something is off with indents here.

Plus: any chances to know the reasoning behind Spring Boot has removed management for this dep?
Perhaps there is something "in favor of", which we should consider ourselves as well.